src/appleseed-max-impl/appleseedoslplugin/oslmaterial.cpp

```diff
@@ -87,8 +87,8 @@ OSLMaterial::OSLMaterial(Class_ID class_id, OSLPluginClassDesc* class_desc)
     m_texture_id_map = m_shader_info->m_texture_id_map;
     m_submaterial_map = m_shader_info->m_material_id_map;
     
-    auto tn_vec = m_shader_info->find_param("Tn");
-    m_has_bump_params = tn_vec != nullptr;
+    m_has_bump_params = m_shader_info->find_maya_attribute("normalCamera") != nullptr;
```

Comment:
Are you sure about this? The `normalCamera` attribute indicates whether there are _bump_ params?

Reply:
Yes, `in_bump_normal` is marked with that tag. In standard_surface it's `in_bump_normal_substrate` so I thought it's easier to find the parameter if they are marked with the same tag..